Codecademy/gamut
# feat(IconButton): aria-describedby override

### Overview


Add override to have IconButton tip optionally not be split into aria-label and described-by (non-breaking)


### PR Checklist

- [x] Related to JIRA ticket: [[gm-788](https://skillsoftdev.atlassian.net/browse/GM-788)]

#### Testing Instructions



* Go to ToolTip page and see new example.
* On VO new and old examples should correctly announce their tip text.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/37945)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/6846)  | [Portal Env](http://www.google.fr/ 'Named link title')   |
